packages/react/package.json

```diff
@@ -23,19 +23,19 @@
     "@babel/preset-react": "^7.25.9",
     "@babel/preset-typescript": "^7.26.0",
     "@iframe-resizer/child": "^5.3.2",
-    "@storybook/addon-a11y": "7.6.20",
-    "@storybook/addon-actions": "7.6.20",
-    "@storybook/addon-essentials": "7.6.20",
+    "@storybook/addon-a11y": "8.4.4",
```

Comment:
Aha, deze mag je allemaal terugdraaien. v8 is nog no go ivm #2316

Reply:
done